docs/content/latest/contribute/docs/docs-page-structure.md

```diff
@@ -50,25 +66,21 @@ menu:
 | `showAsideToc`  | `false` | Should the TOC on the right be enabled |
```

Comment:
| `isTocNested`  | `false` | Should sub-sections be displayed in the TOC on the right. Set to true. (Content pages only) |